src/webpack/index.ts

```diff
@@ -141,10 +155,10 @@ export function getWebpackPlugin<UserOptions = {}> (
         }
 
         // load hook
-        if (plugin.load && plugin.__vfsModules) {
+        if (plugin.load) {
           compiler.options.module.rules.push({
-            include (id) {
-              return id != null && plugin.__vfsModules!.has(id)
+            include () {
+              return true
             },
```

Comment:
I'm not entirely sure whether this change is alright. Would love some more context on why we have that include condition here. Simply having `return true` here makes it consistent with how rollup calls the load hook.